test/e2e/k8s/main_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the License);
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an AS IS BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build integ
+// +build integ
+
+package mcp
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+
+	"github.com/openshift-service-mesh/federation/test/e2e/common"
+)
+
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite(m).
+		Setup(common.CreateControlPlaneNamespace).
+		Setup(common.CreateCACertsSecret).
+		Setup(common.DeployControlPlanes("k8s")).
```

Comment:
There's a couple of other places in this file and the mcp suite file where these constants can be used.
```suggestion
		Setup(common.DeployControlPlanes(config.ConfigModeK8s)).
```

Reply:
Done.